hack/golangci-lint.sh

```diff
@@ -3,6 +3,23 @@
 # Need to run linter twice to cover all the build tags code paths
 set -e
 
+# Mac's are "special" and don't support `declare -A`
+if [[ $(uname -s) == "Darwin" ]] || [[ "$GOOS" == "darwin" ]]; then
+  declare -a DARWIN_SKIP_DIRS
+  DARWIN_SKIP_DIRS=(
+    pkg/api
+    pkg/domain/infra/abi
+    test
+    pkg/machine/qemu
+    libpod/events
+    pkg/trust
+  )
+  echo "Skipped directories: ${DARWIN_SKIP_DIRS[*]}"
+  ./bin/golangci-lint run --build-tags="remote" \
+    --skip-dirs=$(tr ' ' ',' <<<"${DARWIN_SKIP_DIRS[@]}")
+  exit 0  # All done, don't execute anything below
+fi
+
```

Comment:
Comment far, far below, please click the expando arrows once or twice. Line 35: @containers/podman-maintainers, why is there no "remote" in the `for` loop? Does that have anything to do with the `TODO` on line 30?

FWIW I ran `remote` on my laptop and it passed.

Reply:
pkg/api is the backend code for the API. I don't know why it would ever be built when the `remote` tag was specified?

Reply:
I'm not sure how it works, but maybe golangci-lint is not building anything?